src/pages/Trainee/Components/AddDialog/AddDialog.jsx

```diff
@@ -0,0 +1,221 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import * as yup from 'yup';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import PersonIcon from '@material-ui/icons/Person';
+import PropTypes from 'prop-types';
+import EmailIcon from '@material-ui/icons/Email';
+// import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import Grid from '@material-ui/core/Grid';
```

Comment:
Such high import line can be managed using a single import statement
like this:
`import { Button, TextField } from '@material-ui/core';`